### PR TITLE
Experimental

### PR DESCRIPTION
**Pull request by lukrzak**

### CODE REVIEW

**Comment by Shairys** [comment]
lgtm as disscussed in real life

**Review by Shairys — APPROVED** [review]
lgtm as disscussed in real life

File: Car Simulator/Assets/UI/Scripts/ResolutionSetting.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+{
+    private TMP_Dropdown _resolutionDropdown;
+    private Resolution[] _resolutions;
+    private int _currentResolutionIndex;
+
+    public void HandleResolutionChange(int index)
+    {
+        if (_resolutionDropdown != null)
+        {
+
```
Comment (lukrzak) [inline Car Simulator/Assets/UI/Scripts/ResolutionSetting.cs:16]: ?

Comment (kobryl) [reply Car Simulator/Assets/UI/Scripts/ResolutionSetting.cs:16]: > ?

?

**Pull request merged**